elements/drag-drop-elements/pl-drag-drop-element.mustache

```diff
@@ -0,0 +1,51 @@
+{{#question}}
+    <div class="row">
+        <div class="column">
+            <ul id="{{answerName}}-options" name="{{answerName}}" class="connectedSortable">
+                <li class='ui-sortable-handle info info-fixed'>{{header-left-column}}</li>
+                    {{#options}}
+                        <li string="{{.}}">{{.}}</li>
+                    {{/options}}
+                </ul>
+        </div>
+            <div class="column">
+                <ul id="{{answerName}}-dropzone" name="{{answerName}}" class="connectedSortable dropzone">
+                    <li class="ui-sortable-handle info info-fixed">{{header-right-column}}</li>
+                    {{#submission_dict}}
+                        <li string="{{text}}" style='margin-left: {{indent}}px;'>{{text}}</li>
+                    {{/submission_dict}}
+                </ul>
+            </div>
+        </div>
+    <input id="{{answerName}}-input" type="hidden" name="{{answerName}}-input" value=""/>
+
+{{/question}}
+
+{{#submission}}
+    {{^parse-error}}
+        <strong>Your answer: </strong> 
+        <code class="user-output">{{student_submission}}</code>
+        {{#perfect_score}}
+            <span class="badge {{colour}}"><i class="fa fa-check" aria-hidden="true"></i> 100%</span>
+        {{/perfect_score}}
+        {{^perfect_score}}
+                <span class="badge {{colour}}"><i class="fa fa-times" aria-hidden="true"></i>
+        {{score}}%</span> {{feedback}}
+        {{/perfect_score}}
+    {{/parse-error}}
+
+    {{#parse-error}}
+        <strong>Your answer: </strong>
+        <div id="pl-dropdown-{{uuid}}" class="d-inline-block"> 
+            <span class="badge text-danger badge-invalid"><i class="fa fa-exclamation-triangle" aria-hidden="true"></i> Invalid</span>
+            <a href="javascript:void(0);" role="button" class="btn btn-sm btn-secondary small border" data-placement="auto" data-trigger="focus" data-toggle="popover" data-html="true" style="margin-left: 5px" title="Format Error" tabindex="0" data-content="{{parse-error}}"> Why <i class="fa fa-question-circle" aria-hidden="true"></i></a>
+        </div>
+    {{/parse-error}}
+    <div>{{data}}</div>
+{{/submission}}
+
+{{#true_answer}}
+    <strong>Correct answer: </strong> 
+    <code class='user-output'> {{question_solution}} </code> <strong> {{permutationMode}} </strong> 
```

Comment:
Same as above, not all drag-drop blocks are code.

Reply:
this was a suggested styling from @andrewstec, where answers are styled with HTML code blocks. I could change this

Reply:
Maybe we could have a config option so that people who write questions can declare what kind of content is in the block? Maybe options are Math, text, code, etc.?

Its something we can discuss for sure.

Reply:
definitely, I looked at other elements, and it seems like code blocks are only used for code/text answers, and math are rendered as latex elements without blocks

Should I make this a setting then?

Reply:
Maybe, or just leave it as a plain div but apply whatever formatting the question writer had originally applied within the tiles. @nicknytko thoughts?